app/core/src/main/java/com/fsck/k9/Identity.kt

```diff
@@ -0,0 +1,55 @@
+package com.fsck.k9
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Identity(
```

Comment:
:+1: for using `Parcelable` instead of `Serializable`.

But I think we should use `@Parcelize` instead of manually implementing the Parcel (de)serialization.
See https://kotlinlang.org/docs/tutorials/android-plugin.html#parcelable-implementations-generator

Reply:
Ah, I thought that was a thing. Android Studio offered "add parcelable implementation" as a quickfix, which produced this code. Will move to `@Parcelice` :)